baram/efs_manager.py

```diff
@@ -0,0 +1,88 @@
+import boto3
+
+from baram.log_manager import LogManager
+
+
+class EFSManager(object):
+    def __init__(self):
+        self.cli = boto3.client('efs')
+        self.logger = LogManager.get_logger()
+
+    def list_file_systems(self):
+        """
+        List one or more of your EFS.
+
+        :return:
+        """
+        return self.cli.describe_file_systems()['FileSystems']
+
+    def list_mount_targets(self, file_system_id: str):
+        """
+        List mount targets of specific file system.
+
+        :param file_system_id: FileSystemId
+        :return:
+        """
+        try:
+            return self.cli.describe_mount_targets(FileSystemId=file_system_id)['MountTargets']
+        except:
+            self.logger.info('error')
```

Comment:
https://www.geeksforgeeks.org/how-to-print-exception-stack-trace-in-python/

Reply:
Oh I forgot the existence of `traceback`. Thanks for reminding me. This would be applied right away.